016_IP_L151/016_IP_L151_safe.py

```
@@ -0,0 +1,366 @@
+    print(image_path)
+    return send_file(image_path, mimetype='image/png') 
+@app.route('/')  
+def net():  
+    return redirect('http://116.62.60.158/login', code=302) 
+# {/fact}
+
+@app.route('/admin')
+def admin():
+    cookie_1 = request.cookies.get("ss")
+    if cookie_1 == 'admin':
+        return render_template(f'admin.html')
```

Ruff (0.14.1)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)